staffing/views.py

```diff
@@ -1027,38 +1034,43 @@ def all_timesheet(request, year=None, month=None):
     consultants = Consultant.objects.filter(id__in=consultants).order_by("name")
     missions = sortMissions(Mission.objects.filter(id__in=missions))
     charges = {}
+
     if "csv" in request.GET:
         # Simple consultant list
         data = list(consultants)
     else:
-        # drill down link
-        data = [mark_safe("<a href='%s?year=%s;month=%s;#tab-timesheet'>%s</a>" % (reverse("people:consultant_home", args=[consultant.trigramme]),
+        data = [mark_safe("<a href='%s?year=%s;month=%s;#tab-timesheet' class='pydici-tooltip' title='%s'>%s</a>" % (reverse("people:consultant_home", args=[consultant.trigramme]),
                                                                                    month.year,
                                                                                    month.month,
-                                                                                   escape(str(consultant)))) for consultant in consultants]
-    data = [[_("Mission"), _("Mission id")] + data]
+                                                                                   escape(unicode(consultant.name)),
```

Comment:
no more unicode() in python3. It should be str().

Reply:
updated!